src/pages/AllProjects.vue

```diff
@@ -5,7 +5,7 @@
       <ol class="timeline">
         <single-project v-for="project in projects" :key="project._id" :project="project"></single-project>
       </ol>
-      <v-btn primary dark router to="/projects/new" class="add-new-btn">Dodaj</v-btn>
+      <v-btn light router to="/projects/new" class="blue add-new-btn">Dodaj</v-btn>
```

Comment:
W REACT-cie dobrą praktyką jest, by wszelkie klasy pisać w komponencie, a tutaj piszesz już bezpośrednio na komponencie, używając go w PAGES :P Tu chodzi o to, że w komponencie masz już jakieś klasy zdefiniowane (nie chce mi się sprawdzać :P ) i tych nie chciałeś tam dodać, bo mają się tylko tutaj pojawić? :D